mysql/metadata.csv

```diff
@@ -42,3 +42,108 @@ mysql.replication.seconds_behind_master,gauge,,second,,The lag in seconds betwee
+mysql.innodb.mem_dictionary,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_dictionary
+mysql.innodb.mem_total,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_total
+mysql.innodb.os_log_pending_fsyncs,gauge,,operation,,"Number of pending InnoDB log fsync (sync-to-disk) requests.",0,mysql,mysql innodb os_log_pending_fsyncs
+mysql.innodb.os_log_pending_writes,gauge,,write,,"Number of pending InnoDB log writes.",0,mysql,mysql innodb os_log_pending_writes
+mysql.innodb.os_log_written,gauge,,byte,second,"Number of bytes written to the InnoDB log.",0,mysql,mysql innodb os_log_written
+mysql.innodb.pages_created,gauge,,page,second,Number of InnoDB pages created.,0,mysql,mysql innodb pages_created
+mysql.innodb.pages_read,gauge,,page,second,Number of InnoDB pages read.,0,mysql,mysql innodb pages_read
+mysql.innodb.pages_written,gauge,,page,second,Number of InnoDB pages written.,0,mysql,mysql innodb pages_written
+mysql.innodb.rows_deleted,gauge,,row,second,Number of rows deleted from InnoDB tables.,0,mysql,mysql innodb rows_deleted
+mysql.innodb.rows_inserted,gauge,,row,second,Number of rows inserted into InnoDB tables.,0,mysql,mysql innodb rows_inserted
+mysql.innodb.rows_read,gauge,,row,second,Number of rows read from InnoDB tables.,0,mysql,mysql innodb rows_read
+mysql.innodb.rows_updated,gauge,,row,second,Number of rows updated in InnoDB tables.,0,mysql,mysql innodb rows_updated
+mysql.innodb.s_lock_os_waits,gauge,,,second,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output",0,mysql,mysql innodb s_lock_os_waits
+mysql.innodb.s_lock_spin_rounds,gauge,,,second,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb s_lock_spin_rounds
+mysql.innodb.s_lock_spin_waits,gauge,,,second,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb s_lock_spin_waits
+mysql.innodb.x_lock_os_waits,gauge,,,second,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb x_lock_os_waits
+mysql.innodb.x_lock_spin_rounds,gauge,,,,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb x_lock_spin_rounds
+mysql.innodb.x_lock_spin_waits,gauge,,,,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb x_lock_spin_waits
+mysql.galera.wsrep_local_recv_queue_avg,gauge,,,,"Shows the average size of the local received queue since the last status query.",0,mysql,mysql galera wsrep_local_recv_queue_avg
+mysql.galera.wsrep_flow_control_paused,gauge,,percent,,"Shows the fraction of the time, since FLUSH STATUS was last called, that the node paused due to Flow Control.",0,mysql,mysql galera wsrep_flow_control_paused
```

Comment:
```suggestion
mysql.galera.wsrep_flow_control_paused,gauge,,fraction,,"Shows the fraction of the time, since FLUSH STATUS was last called, that the node paused due to Flow Control.",0,mysql,mysql galera wsrep_flow_control_paused
```

Reply:
Great 🙇